dashbuilder/dashbuilder-shared/dashbuilder-js/packages/heatmap-component/src/index.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import * as ComponentDev from "@dashbuilder-js/component-dev";
+import * as ComponentAPI from "@dashbuilder-js/component-api";
+import { SVGHeatmapComponent } from "./SVGHeatmapComponent";
+
+ReactDOM.render(
+  <SVGHeatmapComponent controller={ComponentAPI.getComponentController()} />,
+  document.getElementById("app")!
+);
+
+if (ComponentDev) {
+  ComponentDev.start();
+}
```

Comment:
Hello Tiago!

Thanks for the suggestion. Would you please provide an example of how it could look like? We can refactor for this PR if the change is small.

Thanks!

Reply:
Well, instead of `ComponentAPI.getComponentController()" you could have something like "const componentApi = new ComponentApi() and then componentApi.getController()"